### PR TITLE
Resources

### PR DESCRIPTION
**Pull request by PrateekGupta1509**
- Added estimation resources for Gases emission, Calories Calculator, CO2 absorption Calculator by tree species.
- Added docs for research work done on approximating CO2 sequestered by tree based on height and diameter.
- Added sources for information shown in tiles of knowMore.html

### CODE REVIEW

**Comment by Ceilican** [comment]
Thanks! Could you solve the conflicts, so that I can automatically merge this PR?

**Comment by PrateekGupta1509** [comment]
@Ceilican you can merge it now. Also now that I have added the sources we can start working on reformulating the information and also discuss info that we have to put in contact us tab.

**Comment by Ceilican** [comment]
Thanks!

File: Source/Core/pages/js/caloriesCalc.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
- * Sources:
- * Calories Calculator: http://www.ilovebicycling.com/how-many-calories-do-you-burn-when-cycling/
- * Benfits Source:
- * http://www.sustrans.org.uk/what-you-can-do/use-your-car-less/walking
- * http://www.tmr.qld.gov.au/Travel-and-transport/Cycling/Benefits.aspx
- * http://www.sustrans.org.uk/what-you-can-do/use-your-car-less/health-benefits-walking-and-cycling
- * http://livehealthy.chron.com/sedentary-lifestyle-calorie-intake-3186.html
- */
-
+<<<<<<< HEAD
```
Comment (Ceilican) [inline Source/Core/pages/js/caloriesCalc.js]: Oops!! This line should be removed!

Comment (PrateekGupta1509) [inline Source/Core/pages/js/caloriesCalc.js]: Sorry, I missed that line would remove it along with next PR.

**Pull request merged**